cpp/src/sampling/detail/sampling_utils_impl.cuh

```diff
@@ -18,10 +18,12 @@
 
 #include <cugraph/detail/decompress_edge_partition.cuh>
 #include <cugraph/edge_partition_device_view.cuh>
+#include <cugraph/graph_mask_functions.cuh>
```

Comment:
I feel like it is better to roll-back the mask updates in the sampling code. These sampling code should be re-written with a selection primitive. AFAIK, there is no immediate user to use this masking support for sampling. So, this work will likely to be discarded in the future without actually being used. @ChuckHastings What do you think?

In this PR, we may better focus on defining the graph_mask_t and graph_mask_view_t classes and attaching masks to a graph_view object.